index.html

```diff
@@ -229,38 +229,71 @@ <h3>
-credentials. Concrete serializations will be provided based on <a
-href="https://w3c.github.io/vc-data-model/#json-web-token">VC-JWT</a>, the <a
-href="https://w3c-ccg.github.io/lds-ed25519-2020/">Ed25519 Cryptosuite</a>, 
-the <a href="https://w3c-ccg.github.io/lds-ecdsa-secp256k1-2019/">Secp256k1
-Cryptosuite</a>, and for <a href="https://github.com/w3c-ccg/lds-jws2020/">
-JSON Web Signature 2020</a>. Concrete serializations based on the <a
-href="https://w3c-ccg.github.io/ldp-bbs2020/">BBS+ Cryptosuite</a> might be provided, if an <a
-href="https://identity.foundation/bbs-signature/">IETF BBS+ RFC</a> is published
-in time. Concrete serializations for VC-JWP might be provided, if an <a
-href="https://json-web-proofs.github.io/json-web-proofs/">IETF JWP RFC</a> is
-published in time. 
-The final output specifications might not map directly to
-the list of input drafts. Other output serializations covering cryptographic
-primitives published in IETF RFCs not mentioned in this paragraph might be
-produced.
+This family of specifications defines how to express proofs of integrity for
+verifiable credentials using a number of concrete serializations for each of the
+defined syntaxes. The specific set of concrete serializations included will be
+determined by the Working Group. The following are a non-exhaustive selection
+of expected input documents:
               </p>
-
-              <p class="draft-status"><b>Draft state:</b>
-                <a href="https://w3c-ccg.github.io/data-integrity-spec/">Data Integrity</a>,
+              <p class="input-documents"><b>Container Formats:</b>
                 <a href="https://w3c.github.io/vc-data-model/#json-web-token">VC-JWT</a>,
-                <a href="https://w3c-ccg.github.io/lds-ed25519-2020/">Ed25519 Cryptosuite</a>,
-                <a href="https://w3c-ccg.github.io/lds-ecdsa-secp256k1-2019/">Secp256k1 Cryptosuite</a>,
-                <a href="https://w3c-ccg.github.io/ldp-bbs2020/">BBS+ Cryptosuite</a>,
-                <a href="https://github.com/w3c-ccg/lds-jws2020/">JSON Web Signature 2020</a>              
+                <a href="https://w3c-ccg.github.io/data-integrity-spec/">Data Integrity</a>
+              </p>
+              <p class="input-documents"><b>Cryptosuites:</b>
+                <a href="https://github.com/w3c-ccg/lds-jws2020/">JSON Web Signature</a>,
+                <a href="https://w3c-ccg.github.io/lds-ed25519-2020/">EdDSA</a>,
+                <a href="https://w3c-ccg.github.io/di-ecdsa-secpr1-2019/">NIST ECDSA</a>,
+                <a href="https://w3c-ccg.github.io/lds-ecdsa-secp256k1-2019/">Koblitz ECDSA</a>
+              </p>
               <p class="milestone"><b>Expected completion:</b> <span class=todo>WG-START+24 months</span></p>
             </dd>
           </dl>
 
         </section>
 
+        <section id="normative">
+          <h3>
+            CCG and IETF Specifications
```

Comment:
I am concerned of putting CCG and IETF at the same level. One is a community group that does not produce binding standards with in W3C and the other is the entire SDO that mainly produces binding standards.

also, it does not make sense to be to "produce W3C Recommendations" of the IETF drafts.

Instead of having a separate section CCG and IETF Specifications, I suggest we just simplify data integrity 1.0 deliverable description and classify input documents into three categories: those that are already published, those being worked on in another SDO like IETF and those that maybe worked on in this WG to produce W3C Recommendations (like data integrity).

Reply:
I agree in principle, however I think the fastest path to where we need to be is merging PRs a bit quicker and opening a PR if you have changes you want to see, and less requesting other folks implement your changes in walls of text.

Reply:
> classify input documents into three categories

Ah, I think I see where the miscommunication is happening. Let me try and do the mapping based on what you wrote:

* those that are already published - these are "normative documents that will be referenced from a W3C Recommendation track document".
* those being worked on in another SDO like IETF - these are "normative documents that will be referenced from a W3C Recommendation track document *once they become RFCs*".
*  those that maybe worked on in this WG to produce W3C Recommendations (like data integrity) - these are "input documents"

@Sakurann, does that help clarify where the miscommunication is happening?